api/v1/constructors.go

```diff
@@ -801,7 +801,11 @@ func NewHostProfileSpec(host v1info.HostInfo) (*HostProfileSpec, error) {
 	}
 	spec.Console = &host.Console
 	spec.InstallOutput = &host.InstallOutput
-	spec.MaxCPUMhzConfigured = &host.MaxCPUMhzConfigured
+
+	if host.MaxCPUMhzConfigured != "" {
```

Comment:
Should we need to verify if not nil here?

Reply:
Yes, to avoid invalid entry in config file